modules/ROOT/pages/fips-140-2-compliance-support.adoc

```diff
@@ -51,6 +59,24 @@ image::fips-register-jar-renumber.png[FIPS-register-jar-renumber]
 +
 . Save your changes.
 
+[[example2]]
+==== Example: Setup Instructions Using Bouncy Castle Security Provider in Java 11 and Later
+
+The following example demonstrates how to install and configure Bouncy Castle security provider in Java 11 and later.
+
+. Verify that you are using Java 11 or later and `JAVA_HOME` is set.
+. Download the provider files from https://www.bouncycastle.org/fips-java/[the BouncyCastle web page].
++
+----
+Provider:bc-fips-1.0.2.4  TLS: bctls-fips-1.0.17.jar  PKIX: bcpkix-fips-1.0.7.jar
+----
++
+. Copy the 3 files downloaded to `/lib/boot` (needed for Mule runtime) and `/mule-agent-plugin/lib` (needed for Mule agent) folders in Mule runtime.
+. If you are using CloudHub, run the `amc_setup` parameter and register Mule runtime with CloudHub. For example, `/amc_setup -H d32c4bde-7184-444a-bd87-7a274623e14f---704247 node10  --fips`, where `d32c4bde-7184-444a-bd87-7a274623e14f---704247` is the key from Anypoint Runtime Manager -> Add server.
```

Comment:
What is the meaning of `Add server` here? Is it an option, another step?

Reply:
This would be a question for runtime manager, I have no idea :/

Reply:
@luanamulesoft can we ask an RTM dev?